README.md

```diff
@@ -1,12 +1,13 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-Profile Moblie Verison
+**Profile Moblie Verison**
 
 ### Description of the project.
 
 The Html & Css Mobile View Project is a mimic of figma web templates  
 
+**Portifolio project for mobile view**
```

Comment:
- [OPTIONAL] Please, add a meaningful  `Project description`. Remember A project description is a high-level overview of why you're doing a project. The document explains a project's objectives and essential qualities. Is an essential document that provides your audience with a comprehensive overview of the project. Think of it as the elevator pitch that focuses on what and why without delving into how. 😉 💯 This will make your readme file more informative and professional.